plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -76,6 +76,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                     SUPPORTS_ROW_TYPE,
                     SUPPORTS_SET_COLUMN_TYPE,
                     SUPPORTS_TOPN_PUSHDOWN -> false;
+            case SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT -> true;
```

Comment:
Move this line under `return switch (connectorBehavior) {`

Reply:
Currently, SUPPORTS_AGGREGATION_PUSHDOWN returns false incorrectly and the relevant test is disabled in TestSnowflakeConnectorTest. We should fix it first. Otherwise, we miss some test coverages.